lib/permdeco.gi

```diff
@@ -51,8 +51,12 @@ local   pcgs,r,hom,A,iso,p,i,depths,ords,b,mo,pc,limit,good,new,start,np;
       i:=List(p,x->[(x-A)*(18-x),x]);
       Sort(i);
       p:=i[Length(i)][2]; # best split
-      depths:=Concatenation(Filtered(depths,x->x<=A),[p],
-	        Filtered(depths,x->x>A));
+      if not p in depths then
+	depths:=Concatenation(Filtered(depths,x->x<=A),[p],
+		  Filtered(depths,x->x>A));
+      else
+	good:=false;
+      fi;
     else
```

Comment:
The problem here is that you are mixing tabs and spaces. As a result, the only way to view that code correctly is in an editor with tab width set to 8. Unfortunately, other parts of the code require a tab width of 4 to display correctly... The only sane solution is to use either tabs or spaces, but not to mix them.